src/plugins/manager.js

```diff
@@ -115,7 +118,9 @@ export class PluginPath {
       commands: required.commands && required.commands.map(this.undefaultCommand)
     }
     if (required.type === 'builtin' || /(\\|\/)(src|lib)(\\|\/)commands$/.test(this.path)) return plugin
-    return Namespaces.namespacePlugin(plugin, this.path, this.config)
+    let {namespace} = Namespaces.metaData(this.path, this.config)
+    if (namespace) return Object.assign(plugin, {namespace})
```

Comment:
@RasPhilCo not sure why you need the `Object.assign` here, could probably just be `plugin.namespace = `